dashboard/app/models/aichat_session.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: aichat_sessions
+#
+#  id                   :bigint           not null, primary key
+#  user_id              :integer
+#  level_id             :integer
```

Comment:
`level_id` and `project_id` are sent as strings from the front-end but I see they're integers in `ai_tutor_interaction.rb` so unsure about this.

Reply:
Hmm good callout, I'll make sure that level ID gets stored correctly. And yeah we'll be storing the project_id (integer) rather than the channel_id (string), to your other question